client_cli/pytests/src/client_cli/client_cli.py

```diff
@@ -0,0 +1,255 @@
+"""
+This module contains the ClientCli class, which is responsible for building and executing
+commands for interacting with Iroha blockchain using the Iroha command-line client.
+"""
+import subprocess
+from time import sleep, time
+from typing import Callable
+
+import allure
+
+from src.client_cli.configuration import Config
+
+
+class ClientCli:
+    """
+    A class to represent the Iroha client command line interface.
+    """
+
+    def __init__(self, config: Config, path: str):
+        """
+        Constructs all the necessary attributes for the ClientCli object.
+
+        :param config: The configuration object.
+        :type config: Config
+        :param path: Path to the Iroha client command line interface executable.
+        :type path: str
+        """
+        self.config = config
+        self.default_command = [path, f'--config={self.config.file}']
```

Comment:
Maybe better left as an associated const?
```
class ClientCli:
  DEFAULT_COMMAND = ...

  def __init__(self, ...
```

Reply:
I would also split the path and flags part of the command into it's own field to have less `append`s and `insert`s with arguments to client_cli later, which could be error-prone

Reply:
done